sklearn/neighbors/_subsampled.py

```diff
@@ -0,0 +1,193 @@
+"""Subsampled neighbors transformer"""
+
+# Author: Jennifer Jang <j.jang42@gmail.com>
+#         Heinrich Jiang <heinrichj@google.com>
+#
+# License: BSD 3 clause
+
+import numpy as np
+
+from ..metrics.pairwise import paired_distances, PAIRED_DISTANCES
+from ._base import UnsupervisedMixin
+from ..base import TransformerMixin, BaseEstimator
+from ..utils import check_random_state
+from ..utils.validation import check_is_fitted, _deprecate_positional_args
+from ..utils.validation import check_array
+
+
+class SubsampledNeighborsTransformer(TransformerMixin, UnsupervisedMixin, 
+    BaseEstimator):
+    """Compute subsampled sparse distance matrix of neighboring points in X.
+
+    Parameters
+    ----------
+
+    s : float
```

Comment:
Is there a default value we can use here? If not how would a user decide on what `s` they should use?

Reply:
I've added the default s=0.1 here, since that was a setting we found worked for many datasets.